src/persist/benches/snapshot.rs

```diff
@@ -19,9 +23,47 @@ use persist::file::FileBlob;
 use persist::indexed::Snapshot;
 use persist::mem::MemRegistry;
 use persist::runtime::{self, RuntimeConfig};
-use persist::storage::{Blob, LockInfo};
+use persist::s3::{S3Blob, S3BlobConfig};
+use persist::storage::{Atomicity, Blob, BlobRead, LockInfo};
 use persist::workload::{self, DataGenerator};
 use persist_types::Codec;
+use uuid::Uuid;
+
+pub fn bench_blob_get(data: &DataGenerator, g: &mut BenchmarkGroup<'_, WallTime>) {
+    g.throughput(Throughput::Bytes(data.goodput_bytes()));
+
+    // Only run s3 benchmarks if the magic env vars are set.
```

Comment:
Just double-checking: we only add a bench for S3, not for the regular file backend, while other benches do both?